### ISSUE TITLE
Helppatches

### ISSUE THREAD

**Issue by antongit** [vvvv/VL.StandardLibs#267]
These are still missing correct Filenames, for now these are just the short descriptions.

# Basics
- [ ] Basic Scenegraph Setup: Hello Scenegraph

# Materials
- [x] Reference: Displacement Map from the Skia's Texture, see `Materials\Reference What is Displacement.vl`
- [ ] HowTo DisplacementMap with NormalMap
(**Blocked by:** missing NormalMap TextureFX (Maria has ported the Normal shader from beta for Node20 Materials Workshop, but we were not sure if the Algorithm there is right, so this TextureFX is not included in VL.Stride for now. Should I make a new Issue with the shader attached?)
- [ ] Reference: TesselationMap
- [x] HowTo Material Layers, see `Materials\HowTo Material Layers.vl`

# Meshes
### Dynamic Meshes:

- [x]  2 overlapping noises > Plane + Some spheres, see `Models\Example Dynamic Model from Noise.vl`
- [x]  Noise field, see `Models\Example Poles in the Wind.vl`

- [x] Rutt-Etra Scan Processor with image (https://vimeo.com/468050444), see `Models\Example Texture Readback - Rutt-Etra Lines.vl`
- [ ] Rutt-Etra Scan Processor with live video from MediaFoundation
- [x] Spheres with Tails, see `Models\Example Spheres with Tails.vl`
- [ ] Generative Landsape (DynamicTexture is used for Vertices and Material) (https://derivative.ca/community-post/tutorial/how-make-landscape)
(**Blocked by:** https://github.com/vvvv/VL.StandardLibs/issues/355)

### Instancing 
- [x] Spheres with Instancing (Texture Readback - CPU), see `Models\Example Texture Readback - Spheres with Instancing.vl`
- [ ] Spheres with Instancing and Shaders (GPU only) (like Stride DeepDive workshop)
- [ ] Spheres with Instancing and ShaderFX (GPU only) **(is it possible?)**

### Loading:
- [ ] Loading Mesh (OBJ - LowPoly) and setting Materials 
(**Blocked by:** https://github.com/vvvv/VL.StandardLibs/issues/272, https://github.com/vvvv/VL.StandardLibs/issues/286)

# Lights
- [ ] Reference What Is Skybox Light
- [ ] Reference What Is Ambient Light
- [ ] Reference What Is Point Light
- [ ] Reference What Is Directional Light
- [ ] Reference What Is Spot Light
- [ ] Reference What Is Projector Light
- [ ] HowTo Global Illumination - Light Probes
- [ ] Casting Shadows - Entities and Lights

# PostFX:
- [ ] HowTo Add Ambient Occlusion
- [ ] HowTo Add Local Reflections
- [ ] HowTo Add Depth of Field (DOF)
(**Blocked by:** https://github.com/vvvv/VL.StandardLibs/issues/258)
- [x] HowTo Add Light Streaks
(**Bugs:** https://github.com/vvvv/VL.StandardLibs/issues/257, https://github.com/vvvv/VL.StandardLibs/issues/252)
- [x] HowTo Add Lens Flare
- [ ] HowTo Add Vignette
- [ ] HowTo Add Bloom
- [ ] HowTo Add Film Grain
- [ ] HowTo Set Gamma Correction
- [ ] HowTo Set ToneMap
- [ ] HowTo Set Custom Color Correction
